adr/2024-10-04-product-export-without-manufacturer.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Export products without manufacturer
+date: 2024-10-04
+area: administration
+tags: [administration, product-export]
+---
+
+## Context
+
+Currently twig throws an error if one tries to export a 
+product in a compaaring sales channel.
+
+## Decision
+
+To circumvent this, we render an empty string in this case.
+
+## Consequences
+
+There should be no consequence. Currently we don't create a product import. In the
+future we create a product export without manufacturer, but this is expected
+behaviour if no manufacturer is set.
```

Comment:
I personally do not see the need for an ADR, since it's a simple bugfix. Can you please write a changelog instead?

Reply:
Happy to do! Adr is what I found, @marcelbrode do you have a link „how to changelog“? 😅

Reply:
This applies for all your MR's btw 💙

Reply:
Try `bin/console changelog:create` and you get a little "wizard". If this is based on the ticket, please enter the Ticket number (starting with `NEXT-`), otherwise just use `NEXT-00000` and we will adjust this for you.

More details to be found are here: https://developer.shopware.com/docs/resources/references/adr/2020-08-03-implement-new-changelog.html#implement-new-changelog.

Thank you :)